src/main/java/com/epam/ta/reportportal/dao/util/RecordMappers.java

```diff
@@ -174,7 +175,10 @@
 	};
 
 	public static final RecordMapper<Record, User> USER_MAPPER = r -> {
-		User user = r.into(User.class);
+		User user = r.into(USERS.fieldStream()
```

Comment:
please, make it for all mappers that contain jsonb

Reply:
done